pandas/_libs/tslibs/timezones.pyx

```diff
@@ -28,8 +21,11 @@ from dateutil.tz import (
     tzutc as _dateutil_tzutc,
 )
 import numpy as np
-import pytz
-from pytz.tzinfo import BaseTzInfo as _pytz_BaseTzInfo
+
+try:
```

Comment:
Can we use `import_optional_dependency` here instead?

Reply:
Sure changed